src/wovnio/wovnphp/Utils.php

```diff
@@ -169,4 +175,9 @@ private static function getPath($uri)
         $uri_path = preg_replace("/\?(.)*$/", "", $uri_path);
```

Comment:
Could replace this with removeQueryAndHash. I think it makes sense to remove hash as well.

Reply:
Thx! change this line.